### ISSUE TITLE
Unstable entity_id values

### ISSUE THREAD

**Issue by aschw** [Breina/ha-artnet-led#60]
I've had some issues lately whereby all of the entity ids change.

For example, yesterday I changed the subnet of my local network.   This changed the IP of my Artnet device.  I changed the host IP in configuration.yaml, restarted home assistant, and all of my entitiy IDs changed.  For example, if I used to have an entity id called "light.foo", then I now have an entity ID called "light.foo_2".  light.foo now shows "This entity is no longer being provided by the artnet_led integration. If the entity is no longer in use, delete it in settings."

I don't know if the underlying cause was the IP change or if it was the IP change.

This also happened a couple of months ago.  I think it was a result of updating the version of the add-on and restarting home assistant.  That time, I solved the problem by manually deleting all of the old entities and modifying all of the scenes that referred to them.  This time around, I'm hoping to figure out the underlying cause.

I'm running v0.1.21

Thanks!

**Comment by Breina** [comment]
Yeah sorry, past Breina made the mistake of coupling the unique_id's to the IP. :(

![image](https://github.com/Breina/ha-artnet-led/assets/3009914/12bca1fe-b734-4ba9-824f-345921c63470)

Fixing this will break all existing installations, so I'm holding off fixing this until the next big revision.

**Comment by Breina** [comment]
If have a local DNS, you can use the hostname though, so the underlying IP address is transparent.

**Comment by rtvanhook** [comment]
Anything new here?  I'm having a similar problem after I just repurposed a controller.  I can't seem to shake the old entity IDs.

**Comment by Breina** [comment]
The beta will also include new unique ID's I'm afraid. You will be able to remove the old entities, as they will no longer be referenced by the integration.

https://github.com/Breina/ha-artnet-led/releases/tag/v1.0.0-BETA

**Comment by rtvanhook** [comment]
You got any pointers to migrate?  I'm happy to help test, as well maybe write a migration how-to after I figure it out.  My setup is pretty vanilla, though.  It's architectural, and has a few universes with multiple LED controllers--some RGB and some RGBW--and two universes with pixel controllers.  Nothing fancy like a moving head light.

**Comment by rtvanhook** [comment]
I don't know a good way to abstractly handle the pixel strips.  All I could think of was to write a script to generate the YAML for 100 individual pixels, and then a lighting group for the whole strip.

**Comment by Breina** [comment]
How's this?

https://breina.github.io/ha-artnet-led/migration/
